files/galaxy/tpv/tools.yml

```
@@ -710,10 +710,12 @@ tools:
     env:
       _JAVA_OPTIONS: -XX:MaxPermSize=2G -Xmx6G -Xms1G -Djava.io.tmpdir=/data/2/galaxy_db/tmp -Duser.home=/data/2/galaxy_db/tmp
   toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/.*:
-    env:
-      RM_LIB_PATH: "/data/db/databases/dfam/3.4/"
-  toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.5+galaxy0:
-    cores: 4
+    rules:
+      - if: helpers.tool_version_lt(tool, '4.1.5')
+        env:
+          RM_LIB_PATH: "data/db/databases/dfam/3.4/"
```

```suggestion
          RM_LIB_PATH: "/data/db/databases/dfam/3.4/"
```
I think the leading / was missing

---

Check #739.

---

Sorry for false approval